src/Components/addItem.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import '../App.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
💭 I like that you are importing this CSS so it gets included in our bundle! I think for widespread global css like this, it's best suited in a high-level place in our application, such as `src/index.js` or `src/App.js`, since this style is not specific to only this file. You may want to consider moving this line to one of those files.

Reply:
Thank you @msholty-fd for the detailed feedback.